tests/gdx-tests-android/src/com/badlogic/gdx/tests/android/FragmentTestStarter.java

```diff
@@ -32,7 +33,8 @@ public class FragmentTestStarter extends FragmentActivity implements AndroidFrag
 	@Override
 	protected void onCreate (Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
-		GdxTests.tests.add(MatrixTest.class);
+		if (!GdxTests.tests.contains(MatrixTest.class)) GdxTests.tests.add(MatrixTest.class);
```

Comment:
Does this usually run both MatrixTest and BackTest now? Not a problem, I'm just curious about what changed here.

Reply:
I don't know if the `if` ever serves a purpose, but the regular test starter has it so I made it the same here. And the `BackTest` addition is so that we can run that test in a fragment. It is not automatically run. The only difference now between the available tests in the two Android starters now is `APKExpansionTest`, which I left as a non-fragment exclusive since it just crashes when run.